flake.nix

```diff
@@ -9,60 +9,10 @@
       inputs.nixpkgs-lib.follows = "nixpkgs";
     };
 
-    /*
-      # NOTE: The inputs below this comment are optional
-      # You can remove them with `inputs.<input>.follows = ""`
-
-      For example:
-
-      ```
-      nixvim = {
-        url = "github:nix-community/nixvim";
-        inputs = {
-          nixpkgs.follows = "nixpkgs";
-          flake-parts.follows = "flake-parts";
-
-          devshell.follows = "";
-          flake-compat.follows = "";
-          git-hooks.follows = "";
-          home-manager.follows = "";
-          nix-darwin.follows = "";
-          treefmt-nix.follows = "";
-        };
-      };
-      ```
-    */
-
     nuschtosSearch = {
       url = "github:NuschtOS/search";
       inputs.nixpkgs.follows = "nixpkgs";
     };
-
-    home-manager = {
-      url = "github:nix-community/home-manager";
-      inputs.nixpkgs.follows = "nixpkgs";
-    };
-    nix-darwin = {
-      url = "github:lnl7/nix-darwin";
-      inputs.nixpkgs.follows = "nixpkgs";
-    };
-
-    devshell = {
-      url = "github:numtide/devshell";
-      inputs.nixpkgs.follows = "nixpkgs";
-    };
-    treefmt-nix = {
-      url = "github:numtide/treefmt-nix";
-      inputs.nixpkgs.follows = "nixpkgs";
-    };
-
-    flake-compat.url = "https://flakehub.com/f/edolstra/flake-compat/1.tar.gz";
```

Comment:
I don't think this is an dev input, it's a third kind, only for users who dont't use flakes

Reply:
If it's for users who don't use flakes, then it definitely doesn't belong in the root flake's inputs

Reply:
The root flake should only contain inputs that need to be propagated to end-users' lockfiles